Author: Kevin Chevreuil - Kaisen <kaisen@kaisen-linux.org>

--- lightdm-gtk-greeter-2.0.8.orig/data/lightdm-gtk-greeter.conf
+++ lightdm-gtk-greeter-2.0.8/data/lightdm-gtk-greeter.conf
@@ -1,5 +1,5 @@
 [greeter]
-background = /usr/share/backgrounds/block-black-dragon/3840x2160.png
+background = /usr/share/backgrounds/tobas/3840x2160.png
 theme-name = Kaisen-Dark-Material
 icon-theme-name = Kaisen
 default-user-image = /usr/share/icons/Kaisen/kaisen.png
