Author: Kevin Chevreuil - Kaisen <kaisen@kaisen-linux.org>

--- lightdm-gtk-greeter-2.0.8.orig/data/lightdm-gtk-greeter.conf
+++ lightdm-gtk-greeter-2.0.8/data/lightdm-gtk-greeter.conf
@@ -1,8 +1,9 @@
 [greeter]
-background = /usr/share/backgrounds/dna-left/1920x1080.png
+background = /usr/share/backgrounds/block-black-dragon/3840x2160.png
 theme-name = Kaisen-Dark-Material
-default-user-background = /usr/share/backgrounds/dna-left/1920x1080.png
 icon-theme-name = Kaisen
 default-user-image = /usr/share/icons/Kaisen/kaisen.png
 user-background = false
 font-name = Cantarell 11
+clock-format = %A %d %B %Y : %H:%M:%S
+indicators = ~spacer;~spacer;~host;~spacer;~clock;~power
